### DISCUSSION TITLE
How to configure a different git url?

### DISCUSSION THREAD

**Discussion by ComradeMoustache** [Vinzent03/obsidian-git#138] [Problems]
Hello,

I was wondering how I could prevent obsidian git from using the default github url ("git@github.com") when doing automatic backups? 

I have multiple accounts set up on my computer, due to work, and I have my `~/.ssh/config` set as below:
```
# Default github account: work
Host github.com
   HostName github.com
   IdentityFile ~/.ssh/<private work key>
   IdentitiesOnly yes

# Other github account: personal
Host github-personal
   HostName github.com
   IdentityFile ~/.ssh/<private personal account key>
   IdentitiesOnly yes
```

This means that when I clone my repo I can chose which host to use by specifying either `git clone git@github-personal:denolehov/obsidian-git.git` to use the ssh key of my personal account or `git clone git@github.com:denolehov/obsidian-git.git` to use my work account ssh key. 

I can see that obsidian tries to use `git@github.com` as default which returns `Permission denied (publickey)` whenever it tries to automatically update. I'd like to change that behavior so that it calls `git@github-personal`, is this possible?

Thank you.

**Comment by Vinzent03** [comment]
There's a command to edit the remote urls. Just replace the url with the correct one.
